paper-drawer-panel.html

```diff
@@ -439,7 +440,12 @@
       },
 
       _forceNarrowChanged: function() {
-        this._responsiveChange(this.forceNarrow);
+        if (this.forceNarrow) {
+          this._responsiveChange(true);
+        } else {
+          // set the narrow mode only if we reached the `responsiveWidth`
+          this._responsiveChange(this.$.mq.queryMatches);
+        }
```

Comment:
I would probably just do one-liner ```this._responsiveChange(this.forceNarrow || this.$.mq.queryMatches);``` here.
Minor comment and otherwise LGTM :+1: